config/samples/otel.yaml

```
@@ -0,0 +1,31 @@
+apiVersion: core.spinoperator.dev/v1alpha1
+kind: SpinApp
+metadata:
+  name: otel-spinapp
+spec:
+  image: ghcr.io/spinkube/spin-operator/cpu-load-gen:20240311-163328-g1121986
+  executor: containerd-shim-spin
+  replicas: 1
+  resources:
+    limits:
+      cpu: 500m
+      memory: 500Mi
+    requests:
+      cpu: 100m
+      memory: 400Mi
+
+---
+
+apiVersion: core.spinoperator.dev/v1alpha1
+kind: SpinAppExecutor
+metadata:
+  name: containerd-shim-spin
+spec:
+  createDeployment: true
+  deploymentConfig:
+    runtimeClassName: wasmtime-spin-v2
+    installDefaultCACerts: true
+    otel:
+      endpoint: http://otel-collector-opentelemetry-collector.default.svc.cluster.local:4318
+
+      
```

Nit: I like to make the samples as simple and as relevant to what you're demoing as possible.

```suggestion
apiVersion: core.spinoperator.dev/v1alpha1
kind: SpinApp
metadata:
  name: otel-spinapp
spec:
  image: ghcr.io/spinkube/spin-operator/cpu-load-gen:20240311-163328-g1121986
  executor: containerd-shim-spin
  replicas: 1
---
apiVersion: core.spinoperator.dev/v1alpha1
kind: SpinAppExecutor
metadata:
  name: containerd-shim-spin
spec:
  createDeployment: true
  deploymentConfig:
    runtimeClassName: wasmtime-spin-v2
    installDefaultCACerts: true
    otel:
      endpoint: http://otel-collector-opentelemetry-collector.default.svc.cluster.local:4318
```